### ISSUE TITLE
model 41_BasePreSRG_v3 not 'tuned'

### ISSUE THREAD

**Issue by aaronmberger** [cgrandin/hake-assessment#198]
Providing the model with more flexibility with selectivity also suggested tuning the comps a bit following our normal, McAllister and Ianelli, approach.  Not horrible but r4SS output suggested adjusting the fish comp effective sample size upwards by ~37% and the survey comps down ~9%.  I did that and it didn't make too much difference (in terms of spawning biomass) in the results.  I'm running an mcmc on this model tonight so we can look at/compare the results.  I'm rerunning all the sensitivities now assuming we will use the model 45_BasePreSRG_v4 (which is essentially model 41 with the effective sample sizes of the comps iteratively tuned).  We should talk about this on Monday morning.

**Comment by taylori** [comment]
Hi Aaron,
I woke up too early with hake swimming through my head so I thought I'd check in on email. Thanks for thinking to do the re-tuning. We should make ourselves a checklist of things to update either each year or each time the model changes.

I suspect that increasing the weight given to the fishery comps will stabilize the estimated selectivity a little bit and result in a better model. Great that we should have results to talk about on Monday.

**Comment by aaronmberger** [comment]
Oh no, the dreaded hake wake-mare.  Agree about a list - I had done the tuning earlier so should have spoke up sooner.
My plan for today:
I'm uploading models 45 - 62 now (will take awhile) and will upload model 45 mcmc once its done (have about 4 hours left on it).  I'll commit my changes to modelI-setup.r later today as well as try to build after that.  I'll be working intermittently today, but definitely not between noon and 4PM when my Green Bay Packers play the Atlanta Falcons, winner goes to the super bowl....

**Comment by andrew-edwards** [comment]
I can prob a work a couple of hours this afternoon. Aaron - thanks for doing new models. Commit what you have and I can do the selectivity groupings if you haven't done them. Sounds like you might have already done that! Leave me a note of what I can do if necessary - or I'll just look through some issues.... Enjoy the game.

**Comment by aaronmberger** [comment]
models uploaded, still waiting on mcmc (close)

**Comment by aaronmberger** [comment]
mcmc run uploading now.. .may take a short while to upload

**Comment by cgrandin** [comment]
I've downloaded the new models and have merged the changes that were made to the bridge and sensitivity groups, and it all works. Check out the last few commits to see what I changed to fix the error we were getting on loading. It seems that if a non-base model has an mcmc directory in it, the lapply function causes a 'promise' error which means that an argument name is set to another with the same name somewhere. If we need to have sensitivity models loading their mcmc folders, (i.e. for the selectivity mountains plots) it will be a bit of work and a little ugly but it shouldn't pose too great a problem.

**Comment by aaronmberger** [comment]
Great Chris, I was just going to start diagnosing why mine didn't build last night.  I'll merge and try again.  So did you just move the non-base model mcmc sub-folders out of the models folder then for now?

**Comment by cgrandin** [comment]
G'morning - No, I left them there - better to have it that way. The code now knows to ignore them for the non-base model.

**Comment by aaronmberger** [comment]
Mine built fine with fix, thanks Chris

**Comment by taylori** [comment]
I just finished all.r and am doing build.doc now. I'm up for 10am as suggested in Aaron's email unless somebody wants more time to look at models. I'm going to pause to eat some breakfast before the call.
